source/main/menus/win-linux.js

```diff
@@ -8,9 +8,12 @@ import { environment } from '../environment';
 import { NOTIFICATIONS } from '../../common/ipc/constants';
 import { showUiPartChannel } from '../ipc/control-ui-parts';
 import { generateSupportRequestLink } from '../../common/utils/reporting';
+import { getRtsFlags } from '../utils/rtsFlags';
 
 const id = 'menu';
-const { isWindows, isBlankScreenFixActive } = environment;
+const { isWindows, isBlankScreenFixActive, network } = environment;
+const rtsFlags = getRtsFlags(network);
+const rtsFlagsEnabled = rtsFlags && rtsFlags?.length > 0;
```

Comment:
!!rtsFlags if using the same syntax as elswhere in the codebase,  also second time this check appears, maybe cause for a helper method?

Reply:
Also worth mentioning that if condition would be `>=` then it's `true` for all falsy values.

Probably `!!rtsFlags?.length && rtsFlags.length > 0` is the safest option.